File: Iniciante/1041 - Coordenadas de um Ponto/Python/1041.py
def main():
    x, y = input().split()
    x = float(x)
    y = float(y)

    print(quadrante(x, y))


def quadrante(x, y):
    if x == 0 and y == 0:
        return "Origem"
    elif x > 0 and y > 0:
        return "Q1"
    elif x > 0 and y < 0:
        return "Q4"
    elif x < 0 and y > 0:
        return "Q2"
    elif x < 0 and y < 0:
        return "Q3"
    elif x == 0 and y != 0:
        return "Eixo Y"
    elif y == 0 and x != 0:
        return "Eixo X"


main()

File: Iniciante/1041 - Coordenadas de um Ponto/JavaScript/1041.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split(" ");

  const coordenadas = lines.map((item) => parseFloat(item, 10));

  console.log(quadrante(coordenadas));
}

function quadrante(coordenadas) {
  let x = coordenadas[0];
  let y = coordenadas[1];

  if (x == 0 && y == 0) {
    return "Origem";
  } else if (x > 0 && y > 0) {
    return "Q1";
  } else if (x > 0 && y < 0) {
    return "Q4";
  } else if (x < 0 && y > 0) {
    return "Q2";
  } else if (x < 0 && y < 0) {
    return "Q3";
  } else if (x == 0 && y != 0) {
    return "Eixo Y";
  } else if (y == 0 && x != 0) {
    return "Eixo X";
  }
}

main();
